WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -1874,22 +2401,22 @@ e_britannia = {
 				color = { 255 15 15 }
 				cultural_names = {
 					#Roman Cultures
-					name_list_roman = cn_vinovia
-					name_list_romano_gallic = cn_vinovia
-					name_list_romano_hispanic = cn_vinovia
-					name_list_romano_british = cn_vinovia
-					name_list_romano_african = cn_vinovia
-					name_list_romano_raetian = cn_vinovia
-					name_list_romano_thracian = cn_vinovia
-					name_list_romano_illyrian = cn_vinovia
-					name_list_romano_pannonian = cn_vinovia
-					name_list_dalmatian = cn_vinovia
-					name_list_romano_noric = cn_vinovia
-					name_list_romano_gothic = cn_vinovia
-					name_list_romano_dacian = cn_vinovia
-					name_list_romano_frank = cn_vinovia
-					name_list_romano_aramean = cn_vinovia
-					name_list_romano_vandalic = cn_vinovia
+					name_list_roman = cn_lugdunum
+					name_list_romano_gallic = cn_lugdunum
+					name_list_romano_hispanic = cn_lugdunum
+					name_list_romano_british = cn_lugdunum
+					name_list_romano_african = cn_lugdunum
+					name_list_romano_raetian = cn_lugdunum
+					name_list_romano_thracian = cn_lugdunum
+					name_list_romano_illyrian = cn_lugdunum
+					name_list_romano_pannonian = cn_lugdunum
+					name_list_dalmatian = cn_lugdunum
+					name_list_romano_noric = cn_lugdunum
+					name_list_romano_gothic = cn_lugdunum
+					name_list_romano_dacian = cn_lugdunum
+					name_list_romano_frank = cn_lugdunum
+					name_list_romano_aramean = cn_lugdunum
+					name_list_romano_vandalic = cn_lugdunum
```

Comment:
Vinovia was an attested settlement here, what is Lugdunum based on?

Reply:
> Vinovia was an attested settlement here, what is Lugdunum based on?

https://www.romaneranames.uk/l/lugundun.htm

Reply:
Ok, but that should probably be a barony then, Vinovia is a more sigificant settlement here

Reply:
> Ok, but that should probably be another minor barony then, Vinovia is a more sigificant settlement here

I called the county of Durham Vinovia and barony of Durham Lugdunum. IMO it will show Vinovia's significance while not fully removing Lugdunum as it was a direct predecessor to Durham.

Reply:
In my view Durham becomming later on a major city doesn't mean its previous settlement should be given more importance. Removing Vinovia as a settlement and only representing at county level seems wrong in my view. I agree that Lugdunum can be represented, but not as the main barony here

Reply:
> In my view Durham becoming later on a bigger city doesn't mean its previous settlement should be given more importance. Removing Vinovia as a settlement and only representing at county level seems wrong in my view. I agree with you that Lugdunum can be represented, but not as the main barony here

Ok, I changed it, however I should warn that thes name of Segodunum and Vinovia originally applied only to counties, but not to any baronies.

Reply:
> > In my view Durham becoming later on a bigger city doesn't mean its previous settlement should be given more importance. Removing Vinovia as a settlement and only representing at county level seems wrong in my view. I agree with you that Lugdunum can be represented, but not as the main barony here
> 
> Ok, I changed it, however I should warn that names Segodunum and Vinovia originally applied only to counties, but not to any baronies. Though I personally  prefer minor settlements over major ones if these were direct predecessors to vanilla baronies.